sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/README.md

```diff
@@ -79,9 +79,8 @@ azure:
     client-secret: <client-secret>
     tenant-id: <tenant-id>
     user-group:
-      allowed-group-names: group1,group2
-      allowed-group-ids: <group1-id>,<group2-id>
-      enable-full-list: false  
+      allowed-group-names: <group1>,<group2>
+      allowed-group-ids: <group1-id>,<group2-id>   # When 'all' is used, all group id can be obtained.
```

Comment:
Should we throw exception when `allowed-group-ids: <group1-id>,<group2-id>, all`?